x-pack/plugins/security_solution/public/detections/components/rules/step_define_rule/translations.tsx

```diff
@@ -70,3 +70,24 @@ export const ENABLE_ML_JOB_WARNING = i18n.translate(
       'This ML job is not currently running. Please set this job to run via "ML job settings" before activating this rule.',
   }
 );
+
+export const THREAT_MATCH_INDEX_HELPER_TEXT = i18n.translate(
+  'xpack.securitySolution.detectionEngine.createRule.stepDefineRule.threatMatchingIcesHelperDescription',
+  {
+    defaultMessage: 'Select threat indices',
+  }
+);
+
+export const THREAT_MATCH_REQUIRED = i18n.translate(
+  'xpack.securitySolution.detectionEngine.createRule.stepDefineRule.customThreatQueryFieldRequiredError',
+  {
+    defaultMessage: 'At least one threat match is required.',
+  }
+);
+
+export const THREAT_MATCH_EMPTIES = i18n.translate(
+  'xpack.securitySolution.detectionEngine.createRule.stepDefineRule.customThreatQueryFieldRequiredEmptyError',
+  {
+    defaultMessage: 'All matches require both a field and Threat index field.',
```

Comment:
Is "threat" meant to be capitalized here?

Reply:
No I should change that to be lowercase, thank you.